fulp_modules/toys/code/toys.dm

```diff
@@ -3,17 +3,19 @@
 	desc = "A cheaply made toy. Looks like it need some recharge maybe security can help you"
```

Comment:
is the poor grammar here intentional?

Reply:
As far as I know, its not intentional. The original creator of the plushies don't have English as their first language. I personally really don't like all the spelling errors, I was hoping it would just be 'batong', but people think otherwise.

Reply:
> "A cheaply made toy. Looks like it need some recharge maybe security can help you"

> I feel like this one is better left with the grammar error, as it fits with the original message being incomprehensible.